tyk-gateway-oas-generation-documentation/README.md

```diff
@@ -0,0 +1,34 @@
+# Tyk API Documentation
```

Comment:
Hmmm, this needs a /docs addition. We currently don't have that folder, but, suggesting `/docs/swagger.md` at this point (we may change this in future as conventions get adjusted/considered).

Docs has some issues:

- consistency (semver is `vN.N.N`, several `v` prefixes omitted)...
- language, `add it to it specific tag` - maybe just re-read it or throw it through a grammar check :)
- spacing (L5/headings should follow with an empty line like L1, 13, 20, 31)